geomesa-kafka/geomesa-kafka-09-datastore/pom.xml

```diff
@@ -67,7 +71,15 @@
         <dependency>
             <groupId>com.google.guava</groupId>
             <artifactId>guava</artifactId>
-            <version>18.0</version>
+        </dependency>
+        <dependency>
+            <groupId>com.googlecode.cqengine</groupId>
+            <artifactId>cqengine</artifactId>
```

Comment:
would these be transitive deps from geomesa-memory anyway? (haven't read that far so maybe not) but if so I would remove them from here.